projects/packages/my-jetpack/_inc/components/close-link/index.js

```diff
@@ -4,10 +4,12 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import styles from './styles.module.scss';
 
-const CloseLink = ( { className } ) => {
+const CloseLink = ( { className, accessibleName } ) => {
 	return (
 		<Link to="/" className={ classNames( styles.link, className ) }>
 			<Icon icon={ close } className={ styles.icon } />
+			{ /* Screen reader users require a textual information of what the button does. */ }
+			<span className={ styles[ 'visually-hidden' ] }>{ accessibleName }</span>
```

Comment:
Just a general question, why do we use additional Span here, and not `aria-label` on the `Link`?

Reply:
Maybe we could also add either default value, `accessibleName = ''` or check and not render `span` if `accessibleName === undefined`?

Reply:
It's equivalent, but the ARIA approach is leaner. I don't know why I didn't go for it :) Thanks!